crates/medea-coturn-telnet/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+//! Implements client to access [Coturn] telnet cli. You can use
+//! [`CoturnTelnetConnection`] directly, but it is recommended to use connection
+//! pool based on [deadpool] that will take care of connection lifecycle.
+//!
+//! [Coturn]: https://github.com/coturn/coturn
+//! [deadpool]: https://crates.io/crates/deadpool
+#![allow(clippy::module_name_repetitions)]
+
+pub mod codec;
```

Comment:
Consider to harden `rustc` lints for this crate. Like `tracerr` has, for example.